app/src/main/kotlin/com/softteco/template/ui/feature/signUp/SignUpScreen.kt

```diff
@@ -125,7 +133,7 @@ private fun ScreenContent(
                 modifier = Modifier.fillMaxWidth()
             ) {
                 Checkbox(
-                    checked = state.termsCheckedStateValue,
+                    checked = state.isTermsAccepted,
```

Comment:
It seems that the T&C checkbox has started to work incorrectly on the signup screen.
1. When all fields are empty, the checkbox cannot be activated.
2. When all the fields are filled in, the checkbox automatically becomes marked. I'm not sure, but it seems to me that the user should mark it manually himself, as it was before.

Reply:
My bad). Fixed